tests/resources/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+enable_testing()
```

Comment:
Why did you decide to place CMakeLists.txt within its own folder resources?
can't it be placed under tests?